libraries/AP_Logger/LogStructure.h

```diff
@@ -1370,9 +1384,10 @@ struct PACKED log_PSCZ {
 // @Field: Safety: Hardware Safety Switch status
 
 // @LoggerMessage: PRX
-// @Description: Proximity sensor data
+// @Description: Proximity Filtered sensor data
 // @Field: TimeUS: Time since system startup
-// @Field: Health: True if proximity sensor is healthy
+// @Field: Layer: Pitch(instance) at which the obstacle is at. Each layer is 30 degrees wide, 0th layer is from - 75 degrees to -45. 2nd layer captures horizontal data.
+// @Field: He: True if proximity sensor is healthy
```

Comment:
This isn't actually correct.  It's incorrect in master.  But healthy comes from the `AP_Proximity_Status` structure:

```
    enum class Status {
        NotConnected = 0,
        NoData,
        Good
    };
```

So "NoData" will also be "True" :-)